src/ansys/meshing/prime/autogen/surfaceutilitystructs.py

```diff
@@ -3275,3 +3275,210 @@ def error_code(self) -> ErrorCode:
+                        n_faces_triangulated if n_faces_triangulated is not None else ( TriangulateResults._default_params["n_faces_triangulated"] if "n_faces_triangulated" in TriangulateResults._default_params else (json_data["nFacesTriangulated"] if "nFacesTriangulated" in json_data else None)))
+        self._custom_params = kwargs
+        if model is not None:
+            [ model._logger.warning(f'Unsupported argument : {key}') for key in kwargs ]
+        [setattr(type(self), key, property(lambda self, key = key:  self._custom_params[key] if key in self._custom_params else None,
+        lambda self, value, key = key : self._custom_params.update({ key: value }))) for key in kwargs]
+        self._freeze()
+
+    @staticmethod
+    def set_default(
+            error_code: ErrorCode = None,
+            n_faces_triangulated: int = None):
+        """Set the default values of TriangulateResults.
+
+        Parameters
+        ----------
+        error_code: ErrorCode, optional
+            Error code associated with failure of operation.
+        n_faces_triangulated: int, optional
+            Number of faces that were triangulated.
+        """
+        args = locals()
+        [TriangulateResults._default_params.update({ key: value }) for key, value in args.items() if value is not None]
+
+    @staticmethod
+    def print_default():
+        """Print the default values of TriangulateResults.
+
+        Examples
+        --------
+        >>> TriangulateResults.print_default()
+        """
+        message = ""
+        message += ''.join(str(key) + ' : ' + str(value) + '\n' for key, value in TriangulateResults._default_params.items())
+        print(message)
+
+    def _jsonify(self) -> Dict[str, Any]:
+        json_data = {}
+        if self._error_code is not None:
+            json_data["errorCode"] = self._error_code
+        if self._n_faces_triangulated is not None:
+            json_data["nFacesTriangulated"] = self._n_faces_triangulated
+        [ json_data.update({ utils.to_camel_case(key) : value }) for key, value in self._custom_params.items()]
+        return json_data
+
+    def __str__(self) -> str:
+        message = "error_code :  %s\nn_faces_triangulated :  %s" % (self._error_code, self._n_faces_triangulated)
+        message += ''.join('\n' + str(key) + ' : ' + str(value) for key, value in self._custom_params.items())
+        return message
+
+    @property
+    def error_code(self) -> ErrorCode:
+        """Error code associated with failure of operation.
+        This parameter is a Beta. Parameter behavior and name may change in future.
+        """
+        return self._error_code
+
+    @error_code.setter
+    def error_code(self, value: ErrorCode):
+        self._error_code = value
+
+    @property
+    def n_faces_triangulated(self) -> int:
+        """Number of faces that were triangulated.
+        This parameter is a Beta. Parameter behavior and name may change in future.
```

Comment:
```suggestion
        """Number of faces that were triangulated.
        This is a beta parameter. The behavior and name may change in the future.
```
I wasn't sure if this property would be looked at before triangulating or after. As a parameter, I suggested "to triangulate" but didn't know if I should also suggest that here.

Reply:
@PipKat this property would be looked at after triangulation.